app/.env.dev

```diff
@@ -1 +1,2 @@
 NEXT_PUBLIC_API_URL=http://localhost:8000
+NEXT_PUBLIC_S3_BUCKET_URL=https://de4ecd9d308a46631d2b677af1d480a0.r2.cloudflarestorage.com/districtr-v2-dev
```

Comment:
(O) do we need to obscure this at all?

Reply:
No – we'll be fetching resources from here so it'll be public regardless, even if we try to obscure it in the FE code w/ a secret. CORS should prevent abuse of the tilesets (once the bucket policy is set to be more stringent, i.e. only accept requests from the FE domain)